content/hardware/05.pro-solutions/solutions-and-kits/opta/tutorials/getting-started-connectivity/content.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Using BLE, WiFi and Ethernet on the Arduino Opta'
```

Comment:
```suggestion
title: 'Using Bluetooth® Low Energy, Wi-Fi® and Ethernet on the Arduino Opta®'
```
Trademark addition to connectivity designations and for Arduino Opta®(Recent addition)

Reply:
Agreed with that.